common/utils.go

```diff
@@ -0,0 +1,23 @@
+package common
```

Comment:
this package include only min functions, maybe rename it to min or math ?

Reply:
added a task